airflow/utils/log/__init__.py

```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
```

Comment:
There is another file under airflow.utils that's named logging.py. Should we change the name of that file?